ui/src/main/webapp/index.js

```diff
@@ -29,13 +31,15 @@ if (process.env.NODE_ENV !== 'production') {
     // use <App /> here rather than require() a <NextApp />.
     try {
       const routes = require('./app').routes
-      render(
+      instance = render(
         <AppContainer errorReporter={({ error }) => { throw error }}>
           <Router history={hashHistory} routes={routes} />
         </AppContainer>,
         document.getElementById('root'))
     } catch (e) {}
   })
+
+  window.forceUpdateThemeing = () => setTimeout(() => deepForceUpdate(instance), 0)
```

Comment:
What was the problem ran into that this is solving? Only asking because the library seemed to say it shouldn't be used production.